mailserver/mailserver_test.go

```diff
@@ -119,25 +128,41 @@ func deliverTest(t *testing.T, server *WMailServer, env *whisper.Envelope) {
 		t.Fatalf("failed to retrieve new key pair with seed %d: %s.", seed, err)
 	}
 	birth := env.Expiry - env.TTL
-	p := &ServerTestParams{
+
+	return &ServerTestParams{
 		topic: env.Topic,
+		birth: birth,
 		low:   birth - 1,
 		upp:   birth + 1,
 		key:   testPeerID,
 	}
-
+}
+func deliverTest(t *testing.T, server *WMailServer, env *whisper.Envelope) {
+	p := serverParams(t, env)
 	singleRequest(t, server, env, p, true)
 
-	p.low, p.upp = birth+1, 0xffffffff
+	p.low, p.upp = p.birth+1, p.birth+1
```

Comment:
Just found this double assignment. Always like single instruction per line more, as you've down 3 lines below.